.tool-versions

```diff
@@ -1,6 +1,6 @@
 awscli latest
 poetry 1.5.1
-jq latest
+jq 1.6
```

Comment:
We might want to leave this as latest. I know you were having issues, but we probably don't want to restrict it.

Reply:
reverted